packages/p2p/src/components/my-profile/my-profile-content.jsx

```diff
@@ -12,6 +12,15 @@ const MyProfileContent = () => {
     const { my_profile_store } = useStores();
     const formik_ref = React.useRef();
 
+    const generatePageHeaderText = () => {
+        if (my_profile_store.should_show_add_payment_method_form) {
+            return localize('Add payment method');
+        } else if (my_profile_store.should_show_edit_payment_method_form) {
+            return localize('Edit payment method');
+        }
+        return localize('Payment methods');
+    };
```

Comment:
Made changes by QA to add `Add payment method` title in responsive screen when user clicks `Add new` and the add payment method form is shown.